lex.go

```diff
@@ -225,6 +225,18 @@ func lexRule(l *lexer) stateFn {
 
 // lexComment consumes a commented rule.
 func lexComment(l *lexer) stateFn {
+	//ignore leading spaces and #
```

Comment:
nit: space after // and capitalize, add full stop '.'

// Ignore leading spaces and #.